### PR TITLE
Milestone 1: Capstone Project 1

### PR DESCRIPTION
**Pull request by dqarias**
This is my first capstone project of Microverse,  It's about **"Save Andean Cat"**. The content of this web page is for educational purpose but centered in a real problem over endangered andean cat.

For this project I followed all the requirements that Microverse gave us.

## Project Requirements

- [ ] Personalize the content of your page. Choose a topic that is different than the one in the original design.

- [ ] Follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: colors, typography, layout.

- [ ] The pages should look almost identical to the original design. 

- [ ] You can use a CSS framework (for example Bootstrap) for styling, if you want to.

- [ ] Build only these 2 pages: home and about page.

- [ ] Each of these pages should have versions for 2 different screen sizes: mobile and desktop.

- [ ] The home page should have a link in the menu to the about page.

- [ ] The logo in the header links to the home page.

- [ ] When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.

- [ ] The mobile menu should have a close (X) button that closes the menu.

- [ ] The section "Featured speakers" should be created dynamically in JavaScript.

- [ ] You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

## Project Documentation

Loom Video: [Watch a video of the project](https://www.loom.com/share/be64d606068141afa1cd8372a9b6ff1c)
Live Demo : [Live Demo Link](https://dqarias.github.io/Capstone1-page/)

## Project Delay

> I apologize for sending the project out of time, the reason is: I decided to use sass and this caused me delays, for the next one I will take precautions and not be in the same situation

### CODE REVIEW

File: scss/components/_image.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.program-img {
+  width: 100%;
+  height: 100%;
+  box-shadow: 1px 1px 20px var(--color-ligth-gray);
+  object-fit: cover;
+}
```
Comment (Wes-Isaac) [inline scss/components/_image.scss]: - [ ] Great job so far, In the design, we can see that the images have a checkered background, to be consistent with the design kindly add them.

### Your Implementation
image

### Behance Design
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        <h2 class="a-about__title">SaveCats.org presents Pata Project</h2>
+        <p class="a-about__description">
+          Pata Project, a SaveCats.org Signature Program, is dedicated to the
+          conservation of species identified as in peril, threatened, endangered
+          or close to extinction throughout the world.
+        </p>
+        <div class="a-about__img">
+          <img class="a-about-img" src="./assets/images/logo-3.png" alt="" />
+        </div>
+      </section>
```
Comment (Wes-Isaac) [inline about.html]: - [ ] Great job so far, in the design we can see that after the logo there is a date and location of the event, kindly add that.

### Your Implementation
image

### Behance Design
image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    </main>
+    <footer class="footer bg__dark">
+      <section class="copyrigth__container">
+        <div class="copyrigth__image">
+          <img class="copyrigth-img" src="./assets/images/logo-3.png" alt="" />
+        </div>
+        <div class="copyrigth__text">
+          <p class="mobile-text">
+            2022 Pata Projects Peru <br /><br />
```
Comment (Wes-Isaac) [inline about.html]: - [ ] You did a great job with the footer, just one thing remaining, for the mobile version of the about page the footer has a white background, let's fix that 👍. 

### Your Implementation
image

### Behance Design
image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @dqarias, 

Good job so far! I really like your project keep rocking 🚀.
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: scss/components/_image.scss (hunk 1 above)
Comment (dqarias) [reply scss/components/_image.scss]: I've Just done it, thanks.

background_images

File: about.html (hunk 2 above)
Comment (dqarias) [reply about.html]: I've Just done it, thanks.

logo-about

File: about.html (hunk 3 above)
Comment (dqarias) [reply about.html]: I've Just done it, thanks.

footer-mobile

**Review by codepantha — APPROVED** [review]
Hi @dqarias 👋,

## STATUS: APPROVED ✔️ 

Beautiful project! You did well implementing all the required changes 👏  
Your project is now complete and there is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights ⭐ 
✔️ Descriptive PR
✔️ Beautiful UI and concept
✔️ Meaningful commit messages
✔️ Linter checks

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**